ui/app/styles/components/sidebar.scss

```diff
@@ -0,0 +1,47 @@
+.sidebar {
+  position: fixed;
+  background: #ffffff;
+  width: 750px;
+  padding: 24px;
+  right: 0%;
+  overflow-y: auto;
+  bottom: 0;
+  top: 112px;
+  transform: translateX(100%);
```

Comment:
A small change might be to have translateX be slightly beyond 100% to prevent shadow from showing up.

For example, looks like your box shadow has a bleed of 16px, so you could do `transform: translateX(calc(100% + 16px))` here to prevent the sidebar from having any visual footprint while closed.

Reply:
I get this error:

```
Error: 100% and 16px have incompatible units.
   ╷
10 │   transform: translateX(100%+16px);
```